observability/opentelemetry-extension/src/main/java/org/wildfly/extension/opentelemetry/extension/OpenTelemetrySubsystemDefinition.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2021 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.extension.opentelemetry.extension;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.PersistentResourceDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
+import org.jboss.as.controller.SimpleResourceDefinition;
+import org.jboss.as.controller.operations.validation.StringAllowedValuesValidator;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+import org.wildfly.extension.opentelemetry.OpenTelemetryConfigurationConstants;
+
+/*
+ * For future reference: https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#jaeger-exporter
+ */
+
+public class OpenTelemetrySubsystemDefinition extends PersistentResourceDefinition {
+
+    private static final String[] ALLOWED_EXPORTERS = {"jaeger", "otlp"};
+    private static final String[] ALLOWED_SPAN_PROCESSORS = {"batch", "simple"};
+
+    public static final String[] MODULES = {
+            "org.wildfly.extension.opentelemetry-extension"
```

Comment:
Although there are a few cases that don't follow this, we should try to put the MP spec extensions under `org.wildfly.extension.microprofile.` as the majority follow that pattern, so `org.wildfly.extension.microprofile.opentelemetry-extension` is better

Reply:
OpenTelemetry, though, is not a MicroProfile spec, so I put it elsewhere. It *may* become an MP spec at some point, and we'll to figure out what means here if/when that happens.

Reply:
Thanks for the clarification